# Apply this patch to generated schema.rs files so they are ignored by Rustfmt
#
#
@@ -1,3 +1,4 @@
+#![cfg_attr(any(), rustfmt::skip)]
 // @generated automatically by Diesel CLI.

 diesel::table! {
